docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: '3'
+services:
+
+  db:
+    image: postgres:10-alpine
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_DB: app_db
+      POSTGRES_USER: app_user
+      POSTGRES_PASSWORD: postgres!secret
+  # Uncomment this if you want to persist the data.
+  # volumes:
+  #   - "./pgdata:/var/lib/postgresql/data"
+    volumes:
+      - ./create_table.sql:/docker-entrypoint-initdb.d/init.sql
+
+  postgrest:
```

Comment:
```suggestion
  postgres:
```

Reply:
I think `postgrest` is correct, it means `postgREST`

Reply:
`db` is the postgre database and `postgrest` is the tool to expose tables as REST api